content/en/docs/ops/integrations/spire/index.md

```diff
@@ -0,0 +1,272 @@
+   Selector         : k8s:pod-uid:63c2bbf5-a8b1-4b1f-ad64-f62ad2a69807
+   Selector         : k8s:sa:istio-ingressgateway-service-account
+   DNS name         : istio-ingressgateway.istio-system.svc
+   DNS name         : istio-ingressgateway-c48554dd6-cff5z
+   {{</ text >}}
+
+{{< warning >}}
+SPIFFE IDs for workloads must follow the Istio SPIFFE ID pattern: spiffe://<trust.domain>/ns/<namespace>/sa/<service-account>
+
+{{< /warning >}}
+
+See the [SPIRE help on Registering workloads](https://spiffe.io/docs/latest/deploying/registering/) to learn how to create new entries for workloads and get them attested using multiple selectors to strengthen attestation criteria.
+
+## Verifying Entries
+
+Confirm that identities were created for the workloads.
+{{< text bash >}}
+$ kubectl exec -i -t $SPIRE_SERVER_POD -n spire -c spire-server -- /bin/sh -c "bin/spire-server entry show -socketPath /run/spire/sockets/server.sock"
+{{< /text >}}
+
+   {{< text >}}
+   Found 1 entry
+   Entry ID         : c8dfccdc-9762-4762-80d3-5434e5388ae7
+   SPIFFE ID        : spiffe://example.org/ns/istio-system/sa/istio-ingressgateway-service-account
+   Parent ID        : spiffe://example.org/ns/spire/sa/spire-agent
+   Revision         : 0
+   TTL              : default
+   Selector         : k8s:ns:istio-system
+   Selector         : k8s:pod-uid:88b71387-4641-4d9c-9a89-989c88f7509d
+   Selector         : k8s:sa:istio-ingressgateway-service-account
+   DNS name         : istio-ingressgateway-c48554dd6-cff5z
+   {{< /text >}}
+
+## Spire Federation
+
+Istio uses different Envoy validation contexts for workload identities and bundles.
+To enable Spire Federation:
+
+1. Consult [Spire Agent SDS configuration](https://github.com/spiffe/spire/blob/main/doc/spire_agent.md#sds-configuration) and set the following 
+SDS configuration values for your Spire Agent configuration file.
+
+| Configuration              | Description                                                                                      | Validation Context |
+| -------------------------- | ------------------------------------------------------------------------------------------------ |--------------------|
+| `default_svid_name`        | The TLS Certificate resource name to use for the default X509-SVID with Envoy SDS                | default            |
+| `default_bundle_name`      | The Validation Context resource name to use for the default X.509 bundle with Envoy SDS          | null               |
+| `default_all_bundles_name` | The Validation Context resource name to use for all bundles (including federated) with Envoy SDS | ROOTCA             |
+
+This will allow Envoy to get federated bundles from Spire.
+
+2. In your workload deployment, add the pod annotation `spiffe.io/federatesWith`, specifying the trust domain you want the pod federated with:
+
+
+   {{< text  >}}
+   podAnnotations:
+     spiffe.io/federatesWith: "domain.test"
+   {{< /text >}}
+
+## Cleanup Spire
+

```

Comment:
Could we clarify that this is for cleaning up the quick start deployment of SPIRE, the custom SPIRE installation, or both? For example, if it's just for the quick start, change the "Remove created..." line to:

If you installed SPIRE using the quick start SPIRE deployment provided by Istio, use the following commands to remove those Kubernetes resources: